### ISSUE TITLE
App Theme Dark is not really Dark

### ISSUE THREAD

**Issue by Ash258** [wavebox/waveboxapp#824]
* Wavebox Version: 4.3.0
* Operating System & Version: Windows 10 1803 17134.320

It's nice to have option to set dark theme, but this option should be respected in whole UI not just sidebar (Settings, context menu, ...)

All light themes colorizations (Navy, Teal, ..) should have dark alternative and buttons have same color (Navy, Teal, ...)

**Additional info**

Few fast edited css properties, without borders, anything, just fast mockup.

![Proof](https://i.imgur.com/SF9h9RH.png)

**Comment by Thomas101** [comment]
Yep, it's just the sidebars and controls that are normally visible. We'll be extending the theme support to other parts of the UI soon

**Comment by Evertt** [comment]
Yes please use dark reader, that makes everything look the best.

**Comment by kornava** [comment]
whole UI dark theme not planned yet? is it too complicated to implement?
